cmd/kubectl-k8ssandra/register/command.go

```diff
@@ -69,14 +72,16 @@ func NewRegistrationExecutorFromRegisterClusterCmd(cmd cobra.Command) *Registrat
 		destName = registration.CleanupForKubernetes(srcContext)
 	}
 	return &RegistrationExecutor{
-		SourceKubeconfig: cmd.Flag("source-kubeconfig").Value.String(),
-		DestKubeconfig:   cmd.Flag("dest-kubeconfig").Value.String(),
-		SourceContext:    srcContext,
-		DestContext:      cmd.Flag("dest-context").Value.String(),
-		SourceNamespace:  cmd.Flag("source-namespace").Value.String(),
-		DestNamespace:    cmd.Flag("dest-namespace").Value.String(),
-		ServiceAccount:   cmd.Flag("serviceaccount-name").Value.String(),
-		Context:          cmd.Context(),
-		DestinationName:  destName,
+		SourceKubeconfig:   cmd.Flag("source-kubeconfig").Value.String(),
+		DestKubeconfig:     cmd.Flag("dest-kubeconfig").Value.String(),
+		SourceContext:      srcContext,
+		DestContext:        cmd.Flag("dest-context").Value.String(),
+		SourceNamespace:    cmd.Flag("source-namespace").Value.String(),
+		DestNamespace:      cmd.Flag("dest-namespace").Value.String(),
+		ServiceAccount:     cmd.Flag("serviceaccount-name").Value.String(),
+		OverrideSourceIP:   cmd.Flag("override-src-ip").Value.String(),
```

Comment:
issuse: How is this supposed to work when there's no such flag registered in the RegisterClusterCmd? The register above says:

``RegisterClusterCmd.Flags().String("oride-src-ip"``

Thus, using the parameter name ``oride-src-ip`` and this one uses ``override-src-port``to get the value?

Reply:
Oops, looks like you're right indeed. I've pushed a fix here https://github.com/k8ssandra/k8ssandra-client/pull/56